### DISCUSSION TITLE
PHP single string

### DISCUSSION THREAD

**Discussion by teranixbq** [community/community#143839] [Programming Help] [answered]
### Body

how do I turn an array in PHP into a single string with commas separating each item?

### Guidelines

- [x] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by extenv** [comment, marked as answer]
Easy, just use implode(). Here's how it works
$array = ["apple", "banana", "orange"];
$string = implode(", ", $array);
echo $string;

Output :
apple, banana, orange

Thanks me later 👍🏻

**Reply by teranixbq** [reply]
ok thankyou

**Comment by malmorox** [comment]
There is another way that is the same, but with join(), which is an alias of "implode". I leave that option for you to keep in mind:

$array = ["apple", "banana", "orange"];
$string = join(", ", $array);
echo $string;
